digitalpy/core/persistence/impl/default_persistent_object.py

```diff
@@ -1,9 +1,9 @@
-from digitalpy.model.impl.null_mapper import NullMapper
-from digitalpy.core.object_factory import ObjectFactory
-from digitalpy.model.object_id import ObjectId
-from digitalpy.model.persistent_object import PersistentObject
-from digitalpy.model.property_change_event import PropertyChangeEvent
-from digitalpy.model.value_change_event import ValueChangeEvent
+from digitalpy.core.persistence.impl.null_mapper import NullMapper
+from digitalpy.core.util.object_factory import ObjectFactory
+from digitalpy.core.domain.object_id import ObjectId
+from digitalpy.core.persistence.persistent_object import PersistentObject
+from digitalpy.core.persistence.property_change_event import PropertyChangeEvent
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position